contracts/scripts/deploy.js

```diff
@@ -1,24 +1,13 @@
-/**
- * @notice Umbra Deployment script
- * @dev At each contract deployment or transaction, we wait for the transaction to be mined before
- * continuing to the next step. This is done because it makes it simpler to continue from that spot
- * rather than restart the full deployment
- * @dev To initialize a deploy:
- *   `yarn deploy --network <network>`   (where network specifies a netwrk found in the hardhat.config.ts file)
- * If deploying to a local node (--network localhost), first start Hardhat using `yarn hardhat node`
- */
```

Comment:
Why were these comments deleted? If they're outdated we could update them instead

Reply:
I think I removed them in error while moving stuff around. Restored and updated in 9ab9ccf0bcb3a48e38ab6788d369c0df57e4f2a7